docs/test-stop-server.md

```diff
@@ -0,0 +1,38 @@
+#### test-stop-server
+---
+Stop a Liberty server. test-stop-server honors the skipTests, skipITs, and maven-test-skip properties. It also allows you to bypass automatically stopping the server during the post-integration-test phase with pom configuration or a Liberty-specific command line argument.
```

Comment:
add `tick` markdown to code-like elements.

test-stop-server, skipTests, skipITs, maven-test-skip, post-integration-test and any others I missed.